### PR TITLE
NMS-10657: Added %primaryinterface% to events

### PR DESCRIPTION
**Pull request by christianpape**
* JIRA: https://issues.opennms.org/browse/NMS-10657

### CODE REVIEW

File: features/events/daemon/src/main/java/org/opennms/netmgt/eventd/StandardExpandableParameterResolvers.java [hunk 1]
```diff
@@ -238,6 +238,28 @@ public String getValue(String parm, String parsedParm, Event event, EventUtil ev
...
+        }
+
+        @Override
+        public String getValue(final String parm, final String parsedParm, final Event event, final EventUtil eventUtil) {
+            if (event.getNodeid() != null) {
+                try {
+                    return eventUtil.getPrimaryInterface(event.getNodeid());
+                } catch (SQLException ex) {
+                    // do nothing
+                    LoggerFactory.getLogger(getClass()).info("primary interface ipaddr unavailable for node with id:", event.getNodeid(), ex);
```
Comment (j-white) [inline features/events/daemon/src/main/java/org/opennms/netmgt/eventd/StandardExpandableParameterResolvers.java]:
```suggestion
                    LOG.info("primary interface ipaddr unavailable for node with id: {}", event.getNodeid(), ex);
```

Missing `{}` placeholder.

Would also be a good opportunity to refactor the existing `LoggerFactory.getLogger(getClass())` statements to use a static logger instead.

**Comment by j-white** [comment]
> (Looks like there's no good place to document to new expansion option.)

Actually, it looks like there is. Replacement tokens are listed here:
http://docs.opennms.org/opennms/releases/24.1.2/guide-admin/guide-admin.html#_the_eventconf_xml_file_and_its_tributaries

**Comment by indigo423** [comment]
@mfuhrmann As you requested, a docker image for your testing is [here](https://hub.docker.com/r/opennms/horizon/tags)

**Comment by mfuhrmann** [comment]
"Faster than the police allows"  (German idiom)
I'm happy you've done that @christianpape

**Pull request merged**